src/fn/upload-custom-translations.js

```diff
@@ -81,6 +89,34 @@ function parse(filePath, options) {
   });
 }
 
+function checkTranslations(translations, languageCode) {
+  let mf = null;
+  try {
+    mf = new MessageFormat(languageCode);
+  } catch (e) {
+    log.warn(`Cannot check '${languageCode}' translations: ${e.message}`);
+  }
+  let errorsFound = 0;
+  for (const [msgKey, msgSrc] of Object.entries(translations)) {
+    if (!msgSrc) {
+      log.warn(`Empty '${languageCode}' translation for '${msgKey}' key`);
+    } else if (mf) {
+      try {
+        mf.compile(msgSrc);
+      } catch (e) {
+        if (!HAS_MUSTACHE_MATCHER.test(msgSrc)) {
```

Comment:
Nitpick: Is this because mustache format breaks message format? If so, I'd run this test before the compilation step, because there's no point running compile if we're going to ignore any errors, right?

Reply:
> Is this because mustache format breaks message format?

Yes, and checking some repos with translations, we don't mix both systems mustache and messageformat in a single translation.

> there's no point running compile if we're going to ignore any errors, right?

But not all errors are ignored, only if there is a mustache expression.

I can change the code to this, but still looks cumbersome:

```js
    if (!msgSrc) {
      log.warn(`Empty '${languageCode}' translation for '${msgKey}' key`);
    } else if (mf && !HAS_MUSTACHE_MATCHER.test(msgSrc)) {
      try {
        mf.compile(msgSrc);
      } catch (e) {
        log.error(`Cannot compile '${languageCode}' translation ${msgKey} = '${msgSrc}' : ${e.message}`);
        errorsFound++;
      }
    }
```

Reply:
That way looks better to me, but whatever you prefer is fine.